src/api/ddoc/views.rst

```diff
@@ -914,3 +917,45 @@ Sending multiple queries to a view
             }
         ]
     }
+
+.. _api/ddoc/view/pagination:
+
+Pagination
+==========
+
+.. versionadded:: 4.0
+
+There is a support for token based pagination. The token based pagination is an
+alternative to previously recomended :ref:`pagination recipe <views/pagination>`
```

Comment:
[Stylistic] Whether it was a previous recommendation probably isn't relevant here. Might be good detail in a changelog, though.

```suggestion
alternative to :ref:`pagination recipe <views/pagination>`
```